src/zcash/ZcashEngine.ts

```diff
@@ -261,20 +261,19 @@ export class ZcashEngine extends CurrencyEngine<
       ourReceiveAddresses.push(this.walletInfo.keys.publicKey)
     }
 
-    const memos: EdgeMemo[] = []
-    if (tx.memo != null) {
-      memos.push({
+    const edgeMemos: EdgeMemo[] = tx.memos
+      .filter(text => text !== '')
+      .map(text => ({
         type: 'text',
-        value: tx.memo
```

Comment:
The default is `memoName: 'memo'`, which is what we want to show in the GUI. However, we could add this field to be explicit about what this thing is called.

Reply:
Zcash calls them memo so the default is fine

Reply:
changed it anyway